vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/EvalExpressionProcessor.java

```diff
@@ -35,10 +37,11 @@
     private static final Pattern EVAL_PATTERN = Pattern.compile("^eval\\((.*)\\)$", Pattern.CASE_INSENSITIVE
             | Pattern.DOTALL);
     private static final int EVAL_GROUP = 1;
+    private static final Map<String, Object> NAMESPACES = Map.of("math", Math.class, "string", StringUtils.class);
```

Comment:
`"string"` ->. `"stringUtils"`?

`string` could be ambiguous

Reply:
I was thinking about that too, but it looks long and ugly, maybe we could use something shorter? str_ut ?

Reply:
The practice shows that shortenings don't work well after some time, because it's hard to remember them and sometimes it's not enough clear for new users